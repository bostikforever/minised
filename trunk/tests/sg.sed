s/some text/other text/g
