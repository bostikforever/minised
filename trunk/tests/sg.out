This is other text, that should not stay other text.
